layouts/Error.tsx

```diff
@@ -0,0 +1,79 @@
+// CSS
+import styles from '../styles/layouts/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+import React from 'react';
+import { useEffect, useState } from 'react';
+import Footer from '@/components/Footer';
+
+const departure_url = 'https://tcsca';
+
+export default function ErrorLayout({
+  children,
+}:{
+  children: React.ReactNode
+}) {
+  const [allowReport, setAllowReport] = useState(false);
+  useEffect(() => {
+    // If the user is coming from a broken link on our site, they can report an issue
+    if (document.referrer.includes('localhost:3000' || document.referrer.startsWith(departure_url))){
+     setAllowReport(true);
+    }
+    // If not, they don't need to report an issue
+    else{
+      setAllowReport(false);
+    }
+  }, []);
+
+  return(
+    <section className={styles.container}>
+      <NavBar currentPage='Home' />
+      <main className={styles.container}>
+        <div className={styles.page}>
```

Comment:
instead of `stlyes.page` I feel like it might have made sense to take advantage of `scss` nesting here, considering we wont reuse the style. No need to change this just a small code thought.